src/MySql/docs/Update-AzMySqlConfiguration.md

```diff
@@ -9,6 +9,7 @@ schema: 2.0.0
 
 ## SYNOPSIS
 Updates a configuration of a server.
+Use Update-AzMySqlServer instead if you want update AdministratorLoginPassword, sku, etc.
```

Comment:
Having this information in the description is sufficient. Having it in both the synopsis and description seems redundant.
```suggestion
Updates a configuration of a server.
```

Reply:
@mikefrobbins  we need the line of "Use Update-AzMySqlServer instead if you want update AdministratorLoginPassword, sku, etc.". If the statement is not good enough, could you provide some proposals to improve it? Thanks